try {
  const foo = 'a';
} catch (e) {
  console.log(e);
}

console.log('b');
console.log(foo);
